### ISSUE TITLE
JAWS announces link href value on arrows in VC mode in Firefox rather than aria-label

### ISSUE THREAD

**Issue by mraccess77** [FreedomScientific/VFO-standards-support#122]
###summary
JAWS announces a link's  href value on arrows in VC mode in Firefox rather than aria-label -- on tab it is announce correclty.

Sample code
<a data-ng-repeat="item in items" data-ng-href="https://www.facebook.com/" aria-label="Facebook" class="ng-scope" href="https://www.facebook.com/"> 
            <div class="sprite facebook" aria-label="Facebook"></div> </a>

### Expected result
JAWS should announce the aria-label of an anchor on with up/down arrow in firefox like it does in IE and Chrome

### Actual result
The href value is announced on arrows in VC mode.

### Example
Sample code
<a data-ng-repeat="item in items" data-ng-href="https://www.facebook.com/" aria-label="Facebook" class="ng-scope" href="https://www.facebook.com/"> 
            <div class="sprite facebook" aria-label="Facebook"></div> </a>

#### JAWS version and build number
JAWS 2018.1808
 
#### Operating System and version
Windows 7 and 10

#### Browser and version:
Firefox 62

**Comment by mraccess77** [comment]
Related issue -- When the link contains an image with alt text and aria-label on the link -- the alt from the image is announced rather than the aria-label.

**Comment by jducrot** [comment]
@mraccess77 Could you please provide sample code for this issue?
